_overview/fiat-permissions-overview.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title: Permissions in Spinnaker
+order: 210
+published: true
+---
+
+## Overview
+{:.no_toc}
+
+Fiat is the microservice in Spinnaker responsible for authorization (authz) for the other Spinnaker services. By default, it is not enabled, so users are able to perform any action in Spinnaker. This page describes how Fiat interacts with the following Spinnaker services: 
+
+* Clouddriver for account permission
+* Front50 for application permissions
+* Igor for build services permissions
+
+When Fiat is enabled, users start with no permissions and must be explicitly granted permissions. 
+
+For a deeper dive into how authz works for Spinnaker, see [Authorization](https://www.spinnaker.io/setup/security/authorization). Much of Spinnaker's configuration is done through your Halconfig, which can be found in `~/.hal/config`. 
+
+{:.no_toc}
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+## Requirements
+
+To use Fiat, you need an external identity provider. Create the user roles and maintain them in the identity provider. Fiat controls what permissions are mapped to roles.  
+
+Fiat supports the following identity providers:
+* SAML groups (includes OAuth ONLY with OIDC)
+* LDAP
+* GitHub teams
+* Google Groups
+  
+In all these methods, users are referenced by a userId, which is determined by the authentication method of your choice.
+
+You also need access to the Halyard machine in order to configure Spinnaker.
+
+## Clouddriver accounts
+
+Clouddriver is the Spinnaker service that interacts with the various cloud providers. When Fiat is enabled, account permissions for Clouddriver determine whether a role or group can perform the following actions:
+* `READ` - See objects in a given cloud account.
+* `WRITE` - Deploy objects to a given account. 
+
+Note that for AWS, a role/group needs both read and write access to deploy an AMI from the AWS account that Rosco uses to build AMIs.
+
+## Front50 accounts
+
+Front50 is the Spinnaker service that acts as the system of record for all the other Spinnaker services. In other words, all metadata for things such as applications and pipelines are stored in and served by Front50. Control access to Front50 by creating service accounts. This can be done through a series of [cURL commands](https://www.spinnaker.io/setup/security/authorization/service-accounts/).
+
+Service accounts are used to delegate authority to a pipeline to perform actions in Spinnaker. Users with ALL the roles defined in a service account can grant a pipeline “Run as” permission. The service accounts you create should map to roles/groups in your identity provider. Additionally, all pipelines configured to run off of a trigger must also be configured with “Run as” permission, or they will fail.
```

Comment:
I'm not sure if we want to mention this. But there is a flag `fiat.role.orMode` that switches things so a user doesn't need to be in all roles the service account is in to pick that service account. However, if the service account is not carefully configured then this will usually lead to escalation in privilege issue later.

Reply:
I think we should leave this out of this page since this page is meant to be a 10k foot view